core-library/src/main/java/org/silverpeas/core/variables/DefaultVariablesRepository.java

```diff
@@ -0,0 +1,19 @@
+package org.silverpeas.core.variables;
+
+import org.silverpeas.core.persistence.datasource.repository.jpa.NamedParameters;
+import org.silverpeas.core.persistence.datasource.repository.jpa.SilverpeasJpaEntityRepository;
+
+import javax.inject.Singleton;
+import java.util.Date;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Date'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)